sections/TNF.tex

```diff
@@ -89,15 +89,23 @@ \section{Enumerating Turing machines in Tree Normal Form (TNF)}\label{sec:enum}
 
         \end{tikzpicture}
     }
-    \caption{First-level children of the Tree Normal Form (TNF) enumeration of 5-state 2-symbol Turing machines: each node is a Turing machine, nonhalting machines are leaves of the tree. Internal nodes are halting machines, \ie machines eventually reaching an undefined transition (highlighted in magenta) and their children correspond to all the ways to define this undefined transition. By symmetry, at the first level of the TNF tree, we can ignore machines taking a left move. The two halting machines at the first-level of the tree have 12 children each corresponding to all the choices in $\{\szero,\sone\}\times\{\text{R},\text{L}\}\times\{\text{\stateA},\text{\stateB},\text{\stateC}\}$ for defining the magenta transition. Note that, in this case, the choice of states is reduced from $\{\text{\stateA},\text{\stateB},\text{\stateC},\text{\stateD},\text{\stateE}\}$ to $\{\text{\stateA},\text{\stateB},\text{\stateC}\}$ in order to prevent constructing machines that are only a permutation of one-another.}\label{fig:TNF}
+    \caption{First-level children of the Tree Normal Form (TNF) enumeration of 5-state 2-symbol Turing machines: each node is a Turing machine, nonhalting machines are leaves of the tree. Internal nodes are halting machines, \ie machines eventually reaching an undefined transition (highlighted in magenta) and their children correspond to all the ways to define this undefined transition.\protect\footnotemark By symmetry, at the first level of the TNF tree, we can ignore machines taking a left move. The two halting machines at the first-level of the tree have 12 children each corresponding to all the choices in $\{\szero,\sone\}\times\{\text{R},\text{L}\}\times\{\text{\stateA},\text{\stateB},\text{\stateC}\}$ for defining the magenta transition. Note that, in this case, the choice of states is reduced from $\{\text{\stateA},\text{\stateB},\text{\stateC},\text{\stateD},\text{\stateE}\}$ to $\{\text{\stateA},\text{\stateB},\text{\stateC}\}$ in order to prevent constructing machines that are only a permutation of one-another.}\label{fig:TNF}
```

Comment:
Rejected: already covered by " In practice, and in the counts of Table~\ref{tab:TNF-numbers}, leaves of the TNF tree that have all their transitions defined are not enumerated because they are obviously nonhalting -- hence not relevant for computing $S$." 

And this footenote is too big, which brings the risk of disturbing the layout.